src/routers/group.ts

```diff
@@ -42,6 +44,7 @@ group.get(
   '/group/:id',
   paramsIdValidator,
   methodLoggerMiddleware,
+  checkTokenMiddleware,
```

Comment:
You can add it in app.use(), if you add check for URL, as you need this middleware for all endpoints except for the login one.

Reply:
fixed